src/stdpar/plugin-PixelTriplets/HelixFitOnGPU.h

```diff
@@ -47,7 +47,7 @@ class HelixFitOnGPU {
   void launchRiemannKernels(HitsView const *hv, uint32_t nhits, uint32_t maxNumberOfTuples, cudaStream_t cudaStream);
   void launchBrokenLineKernels(HitsView const *hv, uint32_t nhits, uint32_t maxNumberOfTuples, cudaStream_t cudaStream);
 
-  void launchRiemannKernelsOnCPU(HitsView const *hv, uint32_t nhits, uint32_t maxNumberOfTuples);
+  //void launchRiemannKernelsOnCPU(HitsView const *hv, uint32_t nhits, uint32_t maxNumberOfTuples);
```

Comment:
Would
```suggestion
#ifndef DISABLE_RFIT
  void launchRiemannKernelsOnCPU(HitsView const *hv, uint32_t nhits, uint32_t maxNumberOfTuples);
#endif
```
do the same?

Reply:
I actually removed this function definition ins 126abbd46991d384eec8baaf030b76a0d9a7fc1a as it was confusing `nvc++`.

The problem here is that with the `nvcc` split compilation model, `__global__` was redefined in `cudaCompat.h` to work on cpu. With `nvc++`, we can't redefine `__global__`, so the kernel invocation in `launchRiemannKernelsOnCPU` (e.g. `kernelCircleFit<3>`) where causing compilation error as the compiler was expecting standard kernel invocation syntax with `<<<>>>`.